Tests/FilesystemMapTest.php

```diff
@@ -19,8 +19,8 @@ public function setUp()
      */
     public function shouldGetFilesystemByKey()
     {
-        $this->assertInstanceOf('Gaufrette\Filesystem', $this->filesystemMap->get('amazon_fs'), 'should get filesystem object by key');
```

Comment:
Could you revert this change please? The tests fails with lowest version of the lib because `FilesystemInterface` is only available in the latest version of Gaufrette.

Reply:
i fixed the check by enforcing filesystem interface whenever it is available

Reply:
5.3 seems dead? maybe need to remove it

Reply:
Yes I think we'll remove it in the next release of the bundle, same for 5.4 and 5.5 .